subprojects/common/okhttp/src/main/kotlin/com/avito/http/RetryInterceptor.kt

```diff
@@ -23,9 +23,8 @@ public class RetryInterceptor constructor(
     ),
     private val delayMs: Long = TimeUnit.SECONDS.toMillis(1),
     private val useIncreasingDelay: Boolean = true,
-    private val logger: Logger,
-    private val modifyRetryRequest: (Request) -> Request = { it },
-    private val describeRequest: (Request) -> String = { it.url.redact() }
+    private val logger: Logger, // todo remove after release
+    private val modifyRetryRequest: (Request) -> Request = { it }
```

Comment:
Maybe OkhttpClient has some API for modifications. Because we use it only with Elastic

Reply:
it should be applied only to 2+ try inside retry interceptor

Reply:
I want you to notice that `retry` and `modify` are two different intentions, so I don't like this coupling
But feel free to ignore my opinion

Reply:
Wrapper around `RetryInterceptor`, that will add request modification capability should be ok?
I don't know how to modify request after first try, because knowledge about tries encapsulated inside this interceptor

Maybe it could be done using tags. What do you think?

Or you don't understand why anyone could want to modify request on different tries at all?